release-1.8/release_notes_draft.md

```diff
@@ -59,7 +59,15 @@ on supporting the broadest set of workload types, including hardware and perform
 sensitive workloads such as data analytics and deep learning, while delivering
 incremental improvements to node reliability.
 
+### SIG Storage
 
+[SIG Storage](https://github.com/kubernetes/community/tree/master/sig-storage) is
+responsible for storage and volume plugin components. For the 1.8 release, SIG Storage extends
+the Kubernetes storage API beyond just making volumes available to enabling volume resizing
+and snapshotting. Beyond these alpha/prototype features, the SIG, focused on providing users
```

Comment:
For the 1.8 release, SIG Storage extends the Kubernetes storage API beyond  enabling volumes for resizing and snapshotting.

Reply:
@radhikapc this doesn't read correctly to me. This release is not extending beyond `enabling volumes for resizing and snapshotting`.

Reply:
Spoke with @radhikapc offline. Clarification: `making volumes available` == `enable volumes`; but `to enabling volume resizing and snapshotting` is the new feature set and can't be collapsed into the `making volumes available` or `enable volumes`.